tasks/go.py

```diff
@@ -291,7 +291,10 @@ def deps(ctx, verbose=False):
     print("downloading dependencies")
     start = datetime.datetime.now()
     verbosity = ' -x' if verbose else ''
-    ctx.run(f"go mod download{verbosity}")
+    cmd = f"go mod download{verbosity}"
+    for mod in DEFAULT_MODULES.values():
+        with ctx.cd(mod.full_path()):
+            ctx.run(cmd)
```

Comment:
nit
```suggestion
    for mod in DEFAULT_MODULES.values():
        with ctx.cd(mod.full_path()):
            cmd = f"go mod download{verbosity}"
            ctx.run(cmd)
```

Reply:
I moved it out of the loop so we only format the string once, but maybe it's more readable your way :)

Reply:
Or even:
```suggestion
    for mod in DEFAULT_MODULES.values():
        with ctx.cd(mod.full_path()):
            ctx.run(f"go mod download{verbosity}")
```

Reply:
Ah, that's fair, not sure if Python would optimize it. I like your suggestion more (and I think the performance does not matter much here), but it's a nit, so no need to change it if you don't want to

Reply:
Performance doesn't matter here except to satisfy my anxiety for optimization 😛